src/plugins/walletConnectToDapps/components/PageInput.tsx

```diff
@@ -0,0 +1,44 @@
+import { ArrowBackIcon, ArrowForwardIcon } from '@chakra-ui/icons'
+import { HStack, IconButton, Text, useColorModeValue } from '@chakra-ui/react'
+import { FC, useCallback } from 'react'
+
+interface Props {
+  value: number
+  max: number
+  onChange(value: number): void
+}
+
+export const PageInput: FC<Props> = ({ value, max, onChange }) => {
+  const nextPage = useCallback(() => onChange(value + 1), [onChange, value])
+  const prevPage = useCallback(() => onChange(value - 1), [onChange, value])
+
+  return (
+    <HStack
+      align='center'
+      borderWidth={2}
+      borderRadius='md'
+      borderColor={useColorModeValue('blackAlpha.50', 'gray.750')}
+    >
+      <IconButton
+        colorScheme='gray'
+        aria-label='Previous Page'
+        variant='ghost'
+        icon={<ArrowBackIcon />}
+        disabled={value <= 0}
+        style={{ height: 38 }}
```

Comment:
@gomesalexandre how would you think about this? I want this to have the same height as the search bar input (40px). However, since this is wrapped in a HStack with a border, the UI wouldn't be properly aligned. Height 38 makes it align properly

![image](https://user-images.githubusercontent.com/108498243/189547241-abb2744b-b63c-47d5-9e90-ba6c37c3548b.png)

Reply:
Fixed pxs are something to be cautious about but given this looks good both on desktop and mobile, I don't see any issue with it. Summoning @reallybeard for CSS wizardry answer